src/Xamarin.Android.Build.Tasks/MamJsonToXml.cs

```diff
@@ -14,10 +12,10 @@ public class MamJsonToXml : AndroidTask
 		public  override    string  TaskPrefix  => "A2C";
 
 		[Required]
-		public  ITaskItem[] MappingFiles        { get; set; }
+		public  ITaskItem[] MappingFiles        { get; set; } = [];
```

Comment:
Should this also be `= null!`?  It would prevent a GC allocation, and it'll be set to a different value by MSBuild *anyway*…

Reply:
They are the same as `Array.Empty<ITaskItem>()`, it shouldn't allocate.